samples/apps-with-core-angular/tye.yaml

```diff
@@ -0,0 +1,19 @@
+# tye application configuration file
+# read all about it at https://github.com/dotnet/tye
+#
+# when you've given us a try, we'd love to know what you think:
+#    https://aka.ms/AA7q20u
+#
+name: project-tye
+services:
+- name: moviesapi
+  project: MoviesAPI/MoviesAPI.csproj
+  bindings:
```

Comment:
Remove the port?

Reply:
Hi, Do you want me to remove the port? I kept it because angular service will be calling that API on that particular port? angular app port can be removed as its not getting called from any app

Reply:
@davidfowl  Hi, 

You were saying sometime back that you wanted to simplify this further your way. Then, maybe you can merge this and do the required changes on top of it. Otherwise, you can pass on the instructions what you wanted to do. I will accommodate that. Because I don't understand "Remove the port?" thing. If I remove it from .net app, then angular app will fail as it won't be able to discover the port what api is exposing.

Thanks,
Rahul